postgres/README.md

```diff
@@ -400,6 +400,85 @@ Some of the metrics listed below require additional configuration, see the [samp
 
 See [metadata.csv][20] for a list of metrics provided by this integration.
 
+The check does not collect all metrics by default. Configure the following options to enable the respective metrics:
```

Comment:
`collect_database_size_metrics` and `collect_count_metrics` are enabled by default so I think they should be removed from this section